sdk/src/client/api/block_builder/input_selection/transition.rs

```diff
@@ -55,21 +55,12 @@ impl InputSelection {
         // Remove potential sender feature because it will not be needed anymore as it only needs to be verified once.
         let features = input.features().iter().filter(|feature| !feature.is_sender()).cloned();
 
-        let mut builder = AccountOutputBuilder::from(input)
+        let output = AccountOutputBuilder::from(input)
+            .with_mana(0)
```

Comment:
I think we should do this only if it doesn't has an issuer feature, otherwise it's probably annoying to track the account bound mana
And also want to set it to 0 for NFT outputs?

Reply:
block issuer feature *

Reply:
Yes what's I meant :D